docs/webmaster-libraries.md

```diff
@@ -0,0 +1,14 @@
+# Webmaster libraries
+
+Libraries on the "webmaster" plan has extended permissions compared to
+regular libraries. They have access to most of the Drupal
+administration pages, and can install custom modules.
+
+See [Permissions and roles](./permissions-and-roles) for details on
+Drupal setup.
+
+In addition to the Drupal setup, the site should use the `webmaster`
+plan when configured in [DPL
+Platform](https://danskernesdigitalebibliotek.github.io/dpl-docs/DPL-Platform/),
+this creates a `moduletest` environment for testing and implies a
+slightly different deployment scheduling.
```

Comment:
:+1: I appreciate this addition. Consider moving it to `dpl-docs`. 

As I see it the plan handling is handled partly by the CMS and partly with the Platform. Now what we have a general location within the documentation I think this would be a nice place to put this information and then refer to the parts within the individual projects.

With this in mind I also think it would make sense to provide an overview of all the plan types [a bit like we have done elsewhere](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/5bb77ed095e4690871a51fd65694023d62a17a61/docs/configuration-management.md#L14-L14) instead of only mentioning the webmaster libraries.